### DISCUSSION TITLE
TS errors on generated code

### DISCUSSION THREAD

**Discussion by avifatal** [graphql-editor/graphql-zeus#350] [General]
Hi,
I have generated my code and I have types problems, if I add `as any` in `index.ts` it fix the problems.
```
const fetchOptions = options[1] || {} as any'
...


export const Gql = Chain(HOST, {
  headers: {
    'Content-Type': 'application/json',
    ...HEADERS,
  },
} as any);
```
(note the `as any` added to both of the lines).

When I remove the `as any` there are the error:

![image](https://user-images.githubusercontent.com/12430740/202812081-53b5e568-a6f8-4de3-82f5-9740eba700e7.png)

ts version: 4.7.3

What am I missing, I dont think its related to my schema at all.

Thanks

**Comment by aexol** [comment]
install node-fetch I think if you are using node
